UPGRADE-4.2.md

```diff
@@ -5,3 +5,16 @@ Security
 --------
 
  * Using the `has_role()` function in security expressions is deprecated, use the `is_granted()` function instead.
+ * Passing custom class names to the
+   `Symfony\Component\Security\Core\Authentication\AuthenticationTrustResolver` to define
+   custom anonymous and remember me token classes, is deprecated as of 4.2. To
+   use custom tokens, extend the existing `Symfony\Component\Security\Core\Authentication\Token\AnonymousToken`
+   and `Symfony\Component\Security\Core\Authentication\Token\RememberMeToken`.
```

Comment:
and/or?

Reply:
as in "and/or", or replace the and by or?